app/components/CareersPage/Openings.tsx

```diff
@@ -45,12 +45,13 @@ export default function Openings(props: OpeningsProps): JSX.Element {
           direction="column"
           gap={{ base: "3rem", sm: "4rem", "2xl": "6rem" }}
           pt={{ base: "3rem", "2xl": "4rem" }}
-          px={{ base: "1rem", md: "2rem" }}
           minW={{ base: "95vw" }}
         >
           {props.openingsList?.positions?.map((position, i) => (
             <Card
               key={i}
+              width={"90%"}
```

Comment:
Great, thanks for catching that! Did the issue with horizontal scrolling appear only for small screens when you noticed it or for other sizes as well? I could reproduce it only for small screen sizes. I see another unwanted behavior with the cards and I'll get back to it in my next PR, and will address it in my next PR.

Reply:
The horizontal scrolling issue should be fixed for all screen sizes now according to my checks. If you are asking about its behavior before the fix, I'm not sure but think it was only for small screens.